### PR TITLE
Enabled automatic dark mode for dark mode users #28

### LINKED ISSUE #28: ✨ [Feature request]: Enable automatic dark mode for dark mode users
### Is your feature request related to a problem? Please describe.

I use dark mode in my browser because it's better for my eyes and websites with dark mode look quite attractive. However, it would be even nicer if a website enabled a mode by default based on the user's mode settings. This is what I think we should do here as well.

### Describe the solution you'd like.

A CSS selector which looks after the default mode and changes the variable colours to darker shades, as stated on the <code>data-theme</code> selector. It may also require work in the JS selectors to ensure <code>data-theme</code> can be later switched accordingly.

### Describe alternatives you've considered.

A JS script that changes the mode based on user settings.

### Screenshot / Screenshare

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Petrosdevri**
Closes #28.

### CODE REVIEW

**Comment by Petrosdevri** [comment]
I know I was a bit late, because I was busy working on some other projects however here you go.

File: docusaurus.config.js [hunk 1]
```diff
@@ -125,6 +125,11 @@ const config = {
         theme: lightCodeTheme,
         darkTheme: darkCodeTheme,
       },
+      colorMode: {
+        defaultMode: 'light',
+        disableSwitch: true,
```
Comment (Sachin-chaurasiya) [inline docusaurus.config.js:130]: @Petrosdevri , Why we are disabling the theme toggling, what if the user wants to switch from light to dark and vice versa?
```suggestion
        defaultMode: 'light',
  
```

**Review by Sachin-chaurasiya — CHANGES_REQUESTED** [review]

**Comment by Sachin-chaurasiya** [comment]
> I know I was a bit late, because I was busy working on some other projects however here you go.

No hurry @Petrosdevri.

File: docusaurus.config.js (hunk 1 above)
Comment (Petrosdevri) [reply docusaurus.config.js:130]: I didn't know if you wanted that, wait to fix this.

**Review by atapas — APPROVED** [review]

**Pull request merged**